### DISCUSSION TITLE
How long will under construction take?

### DISCUSSION THREAD

**Discussion by dinar2904** [SiimKirjanen/seatreg#54] [General]
![image](https://user-images.githubusercontent.com/95119576/180630339-8d2bbda5-05ac-4633-9223-197ca051a6a9.png)

hi, i need to use this SEATREG plugin, but i see now still under construction.
Please inform to me how long it will take?

Thankyou in advance

**Comment by SiimKirjanen** [comment]
You need to edit the map and save. If you create a new registration then it will be empty be default.
